easybuild/easyconfigs/s/SuperLU_DIST/SuperLU_DIST-9.0.0-foss-2024a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = "EB_SuperLU"
+
+name = 'SuperLU_DIST'
+version = '9.0.0'
```

Comment:
There's a `9.1.0` now, let's bump to that while we're also bumping the PETSc version?